data/images/objects/trampoline/right_trampoline.sprite

```diff
@@ -0,0 +1,19 @@
+(supertux-sprite
+  (action
+    (name "normal")
+    (hitbox 0 2 18 27)
+    (images "right-0.png")
+  )
+  (action
+    (name "swinging")
+    (fps 24)
+    (hitbox 0 2 18 27)
+    (images "right-0.png"
+            "right-1.png"
+            "right-2.png"
+            "right-3.png"
+            "right-4.png"
+            "right-5.png"
+    )
```

Comment:
Is it possible to mirror the left images?
This would avoid the (almost) duplicate image files.

Reply:
I've fixed it as of the latest few commits.

Reply:
thanks